core/src/test/java/org/jsmart/zerocode/core/utils/FieldTypeConversionUtilsTest.java

```diff
@@ -0,0 +1,160 @@
+package org.jsmart.zerocode.core.utils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.JsonPath;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang.text.StrSubstitutor;
+import org.jsmart.zerocode.core.di.provider.ObjectMapperProvider;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.jsmart.zerocode.core.utils.FieldTypeConversionUtils.digTypeCast;
+import static org.junit.Assert.assertEquals;
+
+public class FieldTypeConversionUtilsTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    ObjectMapper mapper = new ObjectMapperProvider().get();
+
+    @Test
+    public void testSubstituted_v3() throws IOException {
```

Comment:
A test in http module please as well as HelloWorld

Reply:
done.